### PR TITLE
chore: package-lock update

### PR DESCRIPTION
**Pull request by breautek**
Solves several (but not all npm audit vulnerabilities) in our dev dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "author": "Apache Software Foundation",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cordova/eslint-config": "file:../cordova-eslint/cordova-eslint-config-3.0.1-dev.tgz",
```
Comment (timbru31) [inline package.json:19]: Is this change intended?

**Review by timbru31 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (breautek) [reply package.json:19]: Nope definitely not. I must have left my environment is a dirty state the last time I touched this repo :sweat_smile: 

At least someone checks the work despite how trivial this is ;)

**Review by erisu — APPROVED** [review]
